### PR TITLE
[WIP] update alluxio instruction on GCP dataproc

### PR DESCRIPTION
**Pull request by mengdong**
[WIP] update alluxio instruction on GCP dataproc

### CODE REVIEW

**Review by sameerz — CHANGES_REQUESTED** [review]

File: docs/get-started/getting-started-alluxio.md [hunk 1]
```diff
@@ -278,6 +278,45 @@ There are two ways to leverage Alluxio in RAPIDS.
...
    ```
 
+## Alluxio in GCP Dataproc w. RAPIDS
+
+1, Create GCP dataproc Alluxio + RAPIDS cluster:
+```
+export REGION=us-central1
+export ZONE=us-central1-b
+export GCS_BUCKET=dongm-alluxio-dataproc
+export CLUSTER_NAME=nvspark-dataproc-alluxio1
```
[the remark this answers is not in the document]
Comment (mengdong) [reply docs/get-started/getting-started-alluxio.md]: fixed

File: docs/get-started/getting-started-alluxio.md [hunk 2]
```diff
@@ -278,6 +278,45 @@ There are two ways to leverage Alluxio in RAPIDS.
...
+export NUM_GPUS=4
+export NUM_WORKERS=4
+
+gcloud dataproc clusters create $CLUSTER_NAME  \
+    --zone $ZONE \
+    --region $REGION \
+    --image-version=preview-ubuntu \
```
[the remark this answers is not in the document]
Comment (mengdong) [reply docs/get-started/getting-started-alluxio.md]: fixed

File: docs/get-started/getting-started-alluxio.md [hunk 3]
```diff
@@ -278,6 +278,45 @@ There are two ways to leverage Alluxio in RAPIDS.
...
+    --master-machine-type n1-standard-8 \
+    --master-boot-disk-size 500 \
+    --num-workers $NUM_WORKERS \
+    --worker-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS \
+    --worker-machine-type n1-highmem-32 \
+    --num-worker-local-ssds 4 \
+    --initialization-actions gs://${GCS_BUCKET}/alluxio-dataproc.sh,gs://goog-dataproc-initialization-actions-${REGION}/gpu/install_gpu_driver.sh,gs://goog-dataproc-initialization-actions-${REGION}/rapids/rapids.sh \
```
[the remark this answers is not in the document]
Comment (mengdong) [reply docs/get-started/getting-started-alluxio.md]: I've added the location but the script needs to be updated

**Review by mengdong — COMMENTED** [review]
add fixes

File: docs/get-started/getting-started-alluxio.md [hunk 4]
```diff
@@ -278,6 +278,45 @@ There are two ways to leverage Alluxio in RAPIDS.
...
+    --optional-components=JUPYTER,ZEPPELIN \
+    --metadata alluxio_root_ufs_uri="gs://${GCS_BUCKET}/tpcds",gpu-driver-provider="NVIDIA",rapids-runtime="SPARK",alluxio_site_properties="alluxio.master.mount.table.root.option.fs.gcs.accessKeyId='GOOGAPHC77BYHEHKXGKQSXSZ';alluxio.master.mount.table.root.option.fs.gcs.secretAccessKey='IFauREQ/tB24o/iUsUw9zPTYxo+2d8Giq0/pPCft'" \
```
[the remark this answers is not in the document]
Comment (mengdong) [reply docs/get-started/getting-started-alluxio.md]: fixed

**Comment by mengdong** [comment]
Close this one and reopen when Alluxio updated their init script

**Pull request closed without merging**